packages/openneuro-components/src/form/Radio.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import './forms.scss'
+
+export interface RadioProps {
+  elementId?: string
+  label: string
+  radioGroupName: string
+  radioNumber: string
+  onClick?: () => void
+  checked: boolean
+}
+
+export const Radio: React.FC<RadioProps> = ({
+  elementId,
+  label,
+  radioGroupName,
+  radioNumber,
+  onClick,
+  checked,
+  ...props
+}) => {
+  return (
+    <span className="custom-radio">
+      <input
+        type="radio"
+        id={elementId}
```

Comment:
Since this `Radio` component mostly functions as a wrapper for the radio input type, I think that the prop names should match the input attribute names. This way, someone coming in that knows how radio buttons work can just assume that the props are like a regular input type radio, and can avoid checking which props go to which attributes. (ie `radioGroupName` -> `name`). I also think `label` and `value` can both get their own props with `label` being used for the label and `value` being used for the button value